BedrockPlugin.cpp

```diff
@@ -46,6 +46,7 @@ BedrockPlugin* BedrockPlugin::getPlugin(const string& name) {
 void BedrockPlugin::enable(bool enabled) { _enabled = enabled; }
 bool BedrockPlugin::enabled() { return _enabled; }
 string BedrockPlugin::getName() { SERROR("No name defined by this plugin, aborting."); }
+const STable& BedrockPlugin::getInfo() { return pluginInfo; }
```

Comment:
They could create their own `MyPluginStatus` command or something, and use that, but I don't see a problem using it here. Plugins are already married to libstuff as part of bedrock anyway.